### ISSUE TITLE
Feedback-Vorschlag (komplex) bei Konflikt-Erkennen-Aufgabe (wäre sicher analog bei Nebenläufigkeit)

### ISSUE THREAD

**Issue by jvoigtlaender** [fmidue/modelling-tasks#270]
**Tutorin:**

- bei 16 a wäre vielleicht eine kleine Erklärung wenn man etwas falsches abgibt hilfreich

![image](https://github.com/user-attachments/assets/4b2f3b09-3dec-463e-a1f4-39c25a95004d)

----

**ich:**

Das ist wahr. Was wäre denn am hilfreichsten? Einfach ein Aufzeigen/Benennen der in Konflikt stehenden Transitionen in der richtigen Antwort? Oder auch für die falsche Antwort eine irgendwie (wie?) geartete Erklärung dafür, dass darin keine Konflikte oder mehr als Paar von in Konflikt stehenden Transitionen vorliegt?

----

**Tutorin:**

vielleicht wäre da diese mathematische Notation hilfreich, um zu zeigen, dass die genannten Transitionen kleiner sind als die Startmarkierung und somit nicht in Konflikt oder so etwas ähnliches

**Comment by marcellussiegburg** [comment]
> vielleicht wäre da diese mathematische Notation hilfreich, um zu zeigen, dass die genannten Transitionen kleiner sind als die Startmarkierung und somit nicht in Konflikt oder so etwas ähnliches

Es sind ja gar keine Transitionen genannt. Vorschlag:
Bei Fehler wird
1. das Petrinetz mit (allen) Bezeichnern erneut gezeichnet
2. es werden alle Transitionen aufgelistet, die nicht aktiviert sind
3. bei den verbleibenden Paaren wird die mathematische Notation genutzt, um zu zeigen, warum kein Konflikt besteht

Nebenläufigkeit analog

(Hier müsste natürlich geschaut werden, welche Informationen bereits zur Verfügung stehen und ggf. ob Alloy die fehlenden Informationen zusätzlich bereitstellen kann.)

**Comment by jvoigtlaender** [comment]
Ja, so in der Art. Wurde auch schon hier diskutiert: https://github.com/jvoigtlaender/modelling-tasks-internal/issues/218

In der Tat war obiges offenbar ein Duplikat von meiner Seite. Hatte die erste Rückmeldung der Tutorin schonmal in dem anderen Issue gepostet. (Vielleicht kam der follow-up Austausch mit ihr erst später zustande.)

Andere Issues, die verwandt und vielleicht sogar Duplikate sind: #254, #255
